pkg/xray/resource_xray_security_policy.go

```diff
@@ -96,6 +111,13 @@ var criteriaMaliciousPkgDiff = func(ctx context.Context, diff *schema.ResourceDi
 	if len(minSeverity) > 0 && len(cvssRange) > 0 {
 		return fmt.Errorf("min_severity can't be set together with cvss_range")
 	}
+	if (len(vulnerabilityIDs) > 0 && maliciousPackage) || (len(vulnerabilityIDs) > 0 && len(minSeverity) > 0) ||
+		(len(vulnerabilityIDs) > 0 && len(cvssRange) > 0) {
+		return fmt.Errorf("vulnerability_ids can't be set together with with malicious_package, min_severity and/or cvss_range")
+	}
+	if len(vulnerabilityIDs) > 99 {
```

Comment:
`> 100`?

Actually the schema `MinItems` and `MaxItems` should prevent this from happening anyway.